### DISCUSSION TITLE
Tidepool Integration Environment Host Change

### DISCUSSION THREAD

**Discussion by toddkazakov** [NightscoutFoundation/xDrip#2870] [General]
Hello,

We are changing the Tidepool integration environment's host from `https://int-api.tidepool.org` to `https://external.integration.tidepool.org`. I think https://github.com/NightscoutFoundation/xDrip/blob/master/app/src/main/java/com/eveningoutpost/dexdrip/tidepool/TidepoolUploader.java#L49 needs to be updated to reflect that change.

Thanks,
Todd

**Comment by Navid200** [comment]
Hi,

Is this not already addressed here?
https://github.com/NightscoutFoundation/xDrip/issues/2597

**Comment by toddkazakov** [comment]
No, the linked issue only deals with the legacy authentication api depreciation.
